packages/EasyCore/src/Bridge/Symfony/ApiPlatform/DataPersister/AbstractContextDataPersister.php

```diff
@@ -0,0 +1,72 @@
+<?php
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Symfony\ApiPlatform\DataPersister;
+
+use ApiPlatform\Core\DataPersister\ContextAwareDataPersisterInterface;
+
+abstract class AbstractContextDataPersister implements ContextAwareDataPersisterInterface
+{
+    /**
+     * @var mixed
+     */
+    protected $dataPersister;
+
+    /**
+     * AbstractContextDataPersister constructor.
+     *
+     * @param mixed $dataPersister
+     */
+    public function __construct($dataPersister)
+    {
+        $this->dataPersister = $dataPersister;
+    }
+
+    /**
+     * Persist the data.
+     *
+     * @param mixed $data
+     * @param null|mixed[] $context
+     *
+     * @return object|void
+     */
+    public function persist($data, ?array $context = null)
+    {
+        return $this->dataPersister->persist($data, $context);
```

Comment:
The api-platform data persister is expecting the context to be an array so we need to default it to an empty array if null.